src/lib/RpcApi.ts

```diff
@@ -22,6 +32,8 @@ import { Raven } from 'vue-raven'; // Sentry.io SDK
 import { ERROR_CANCELED } from './Constants';
 import Config from 'config';
 
+type KeyguardResult = SignatureResult | DerivedAddress[] | SingleKeyResult[] | ExportResult | SimpleResult;
```

Comment:
Sadly there is no combined type export from the Keyguard client yet, so all result types have to be manually combined here.

Reply:
could also be
```suggestion
type KeyguardResult = RedirectResult | ExportResult;
```

`RedirectResult` is supposed to be the combined type but is missing `ExportResult`. I created nimiq/keyguard/pull/383 to include `ExportResult` in `RedirectResult`.